python/main.py

```diff
@@ -40,3 +121,126 @@ async def get_image(items_image):
         image = images / "default.jpg"
 
     return FileResponse(image)
+
+@app.get("/items")
+def get_items():
```

Comment:
for functions that handle requests, I think it's better to add error handling. For example what happen if there's an error reading from database?

Reply:
> for functions that handle requests, I think it's better to add error handling. For example what happen if there's an error reading from database?

Ok, I will add the error handling.